Representable/representable
# submission page thanks modal design

**added thanks modal design to the submission page**

**to test**
- python manage.py runserver
- submit a community
- check thanks modal looks as expected

**note**
- still waiting on hearing about the email automatically sending functionality before updating the wording of this page

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/103420077-bc577780-4b39-11eb-9a5d-ef73dc6d2251.png)
![image](https://user-images.githubusercontent.com/41943646/103420092-c9746680-4b39-11eb-9701-59451a88ec62.png)
